images/hook/entrypoint.sh

```diff
@@ -1,10 +1,25 @@
 #!/bin/sh
 set -e
 
-echo "Assuming changeset from the envrionment: $RIG_CHANGESET"
+echo "Assuming changeset from the environment: $RIG_CHANGESET"
 # note that rig does not take explicit changeset ID
 # taking it from the environment variables
-if [ $1 = "update" ]; then
+if [ $1 = "bootstrap" ]; then
```

Comment:
We don't use rigging for installs. Let's just drop an install script among resources that kubectl's everything, e.g. see https://github.com/gravitational/monitoring-app/blob/version/5.5.x/resources/app.yaml#L23.

Reply:
I thought it was better to stay consistent with the flow... any specific reason to not use rigging during the install phase?

Reply:
Well, for one you'd then also need to handle uninstalls properly, otherwise right now the uninstall hook is broken and the phase can't be rolled back / retried if it fails during install. If you want to use rigging, you need to add support for uninstall here as well and make sure the logging-app install phase can be rolled back / retried. I think it might work, although I'm not 100% as we've never really used rigging for installs (it was designed for upgrades), that's why I suggested to keep things simple.